.github/workflows/nightly_tests.yaml

```diff
@@ -85,7 +85,7 @@ jobs:
       run: python xpk.py cluster delete --cluster $TPU_CLUSTER_NAME --zone=us-central2-b
 
   pw-cluster-and-workload:
-    runs-on: [ubuntu-20.04]
+    runs-on: [ubuntu-22.04]
```

Comment:
There is a bit of code duplication between this job and the job before, some shared steps, but probably not a subject to refactor yet. Just wanted to comment this, feel free to resolve this comment.